index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/1dea6ac3ad.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Manel Hammouche Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <a href="#" id="logo">Manel</a>
+        <i class="fa-solid fa-bars fa-2x" id="menu_icon"></i>
+    </header>
+
+    <section id="headline">
+        <h1>I'm Manel, <br> Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience. If you like what you see and have a project you need coded, don’t hestiate to contact me!
+        </p>
+
+        <h3>LET’S CONNECT</h3>
+        <div id="social_media_links">
+            <a href="https://twitter.com/ha_manel_" target="_blank" rel="noopener" class="social_media_link"><i
+                    class="fa-brands fa-twitter fa-2x"></i></a>
+            <a href="https://www.linkedin.com/in/manel-hammouche/" target="_blank" rel="noopener" class="social_media_link"><i
+                    class="fa-brands fa-linkedin fa-2x"></i></a>
+            <a href="https://github.com/ha-manel" target="_blank" rel="noopener" class="social_media_link"><i
+                    class="fa-brands fa-github fa-2x"></i></a>
+            <a href="https://www.hackerrank.com/manelhammouche3" target="_blank" rel="noopener" class="social_media_link"><i
+                    class="fa-brands fa-hackerrank fa-2x"></i></a>
+        </div>
```

Comment:
- [ ] Kindly use `ul` tag instead of `div` tag and `a` tag should be in `li` tag for each icon here. :writing_hand: